component-api/src/main/java/org/talend/sdk/component/api/service/schema/FixedSchema.java

```diff
@@ -27,8 +27,18 @@
 @Retention(RUNTIME)
 @Documentation("Mark a connector as having a fixed schema. " +
         "Annotation's value must match the name of a DiscoverSchema or a DiscoverSchemaExtended annotation. " +
-        "The related action will return the fixed schema.")
+        "The related action will return the fixed schema for the specified flows.")
 public @interface FixedSchema {
 
     String value() default "";
+
+    /**
+     * The flows to apply the fixed schema.
+     * With an Input component, it will always concern the main flow. So you can omit it.
+     * With an Output component, it will concern either concern the main flow and/or the reject output flows.
```

Comment:
Please, mention it can be also something else that main or reject.

Reply:
Sure!